package/kvmd-fan/kvmd-fan.py

```diff
@@ -3,47 +3,43 @@
 
 import RPi.GPIO
 import time
-RPi.GPIO.setwarnings(False)
-RPi.GPIO.setmode(RPi.GPIO.BCM)
-RPi.GPIO.setup(12, RPi.GPIO.OUT)
-pwm = RPi.GPIO.PWM(12,100)
-RPi.GPIO.setwarnings(False)
 
+#settings
+fan_gpio=12 # Fan GPIO pin number (default: 12)
+idle_speed = 0 # Fan speed when under min_temp(IDLE) (min: 0, max: 100)
+min_temp = 40 # Fan starting temperature 
```

Comment:
It's easy to reach 40 degrees. I suggest you adjust it to 55 degrees.

Reply:
What about 50? 55 is so close from max_temp(60 degrees). Also You can edit this fork 😃